### PR TITLE
Digital Govt Award WRI

### PR DESCRIPTION
**Pull request by BGranberg**
Pull request description
------------------------

### CODE REVIEW

File: _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2016-09-07 19:54:05
+categories:
+
+tags:
+
+---
+
+The updated [Watershed Restoration Initiative (WRI) Application](https://wri.utah.gov/wri/) will be adding a 2016 Digital Government Award to its trophy case this month. WRI was one of 3 winners in the "Driving Digital Government" state government category.({{ "/images/DGAA.png" | prepend: site.baseurl }}){:.inline-text-right} 
+
+The application, built by a partnership between the Division of Wildlife Resources, the Dept. of Technology Services AppDev team, and AGRC, allows for collaborative project proposals, scheduling, funding, tracking, and mapping across the many participating agencies and levels of government. The WRI App integrates GIS-based disgital maps extensively to organize, analyze, and visualize project information.({{ "/images/WRIAppMap2.png" | prepend: site.baseurl }}){:.inline-text-right}
```
Comment (steveoh) [inline _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md]: `disgital` spell check

File: _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+The application, built by a partnership between the Division of Wildlife Resources, the Dept. of Technology Services AppDev team, and AGRC, allows for collaborative project proposals, scheduling, funding, tracking, and mapping across the many participating agencies and levels of government. The WRI App integrates GIS-based disgital maps extensively to organize, analyze, and visualize project information.({{ "/images/WRIAppMap2.png" | prepend: site.baseurl }}){:.inline-text-right} 
+
+To date, large-scale landscape remediation and improvement projects coordinated and tracked in the WRI environment total almost $200 million. Partnering organizations include 6 state agencies, 8 non-profits, and 5 federal agencies, plus, in some cases, private land owners.
+
+The WRI team includes, pictured from left Steve Gourley, Scott Davis, .....John Angus, and Nathan Kota. And, not pictured are: Kiril Boyadjieff, ......({{ "/images/WRITeam.jpg" | prepend: site.baseurl }}){:.inline-text-right}
```
Comment (steveoh) [inline _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md]: haha, the ...'s are great.

File: _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+status: publish
+published: false
+title: 'Watershed-Restoration-Initiative-Wins-2016-Digital-Government-Award'
```
Comment (steveoh) [inline _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md]: you don't want the hyphens in the title

Comment (BGranberg) [inline _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md]: Jessie is going to add the missing names and then I'll let you know and you can make sure its pretty

Comment (steveoh) [inline _posts/2016-09-07-watershed-restoration-initiative-wins-2016-digital-government-award.md]: I've added the names already.

**Comment by steveoh** [comment]
:+1:

**Pull request closed without merging**